README.yaml

```diff
@@ -0,0 +1,60 @@
+---
+#
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+
+# Name of this project
+name: fluentd-datadog-logs
+
+# Logo for this project
+#logo: docs/logo.png
+
+# License of this project
+license: "APACHE2"
+
+# Canonical GitHub repo
+github_repo: cloudposse/fluentd-datadog-logs
+
+# Badges to display
+badges:
+  - name: "Build Status"
+    image: "https://travis-ci.org/cloudposse/fluentd-datadog-logs.svg?branch=master"
+    url: "https://travis-ci.org/cloudposse/fluentd-datadog-logs"
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/fluentd-datadog-logs.svg"
+    url: "https://travis-ci.org/cloudposse/fluentd-datadog-logs/releases"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+related:
```

Comment:
Add this related project: 
https://github.com/cloudposse/charts/tree/master/incubator/fluentd-kubernetes